pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonDefaults.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.extension;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import org.gradle.api.GradleException;
+
+
+public class PythonDefaults {
```

Comment:
The class kind of serves two purposes; it provides default mappings for "Python 2" and "Python 3" (i.e. when a project doesn't care about the minor version, it can just ask for the default Python 3 version).

It also provides the set of Python versions that are allowed to be chosen.  I thought of this as the "default set of allowed Python versions".

`PythonDefaults` felt like a nice succinct name to capture both concepts.  I won't bikeshed on it though if you feel strongly about `SupportedPythonVersions`.